src/types/gecko-profile.js

```diff
@@ -120,12 +121,19 @@ export type GeckoFrameTable = {|
       IndexIntoStringTable,
       // for label frames, whether this frame should be shown in "JS only" stacks
       boolean,
+      // innerWindowID of JS frames. JS frames can be correlated to a Page through this value.
```

Comment:
It might be worth having this comment shorter, and refer to the main processed profile type for the full documentation.